CardParts/src/Extensions/UIView.swift

```diff
@@ -6,6 +6,20 @@
 //
 
 extension UIView {
+    func roundedTop(radius: CGFloat){
```

Comment:
Do we want it to just be `roundedTop` or a more generic `roundCorners` and they pass the corners they want to round?